apps/routes/lib/repo.ex

```diff
@@ -212,7 +215,7 @@ defmodule Routes.Repo do
   @doc """
   The Green Line.
   """
```

Comment:
Does it make sense to move the documentation into `Routes.RepoApi` as well?

Reply:
You're right, I'll re-organize the documentation.